transit_odp/avl/forms.py

```diff
@@ -294,3 +296,235 @@ def get_layout(self):
+        required=False,
+        max_length=256,
+        validators=[length_validator],
+        help_text=_(
+            "This is often the National Operator Code but please check the values used within the feeds you wish to subscribe to"
+        ),
+        error_messages={
+            "invalid": _("Enter an operator ref up to 256 characters"),
+        },
+    )
+
+    vehicle_ref = forms.CharField(
+        label=_("Vehicle ref (optional)"),
+        required=False,
+        validators=[length_validator],
+        help_text=_("Reference to the specific vehicle make a journey or journeys"),
+        error_messages={
+            "invalid": _("Enter a vehicle ref up to 256 characters"),
+        },
+    )
+
+    line_ref = forms.CharField(
+        label=_("Line ref (optional)"),
+        required=False,
+        validators=[length_validator],
+        help_text=_("Name of number by which the line is known to the public"),
+        error_messages={
+            "invalid": _("Enter a line ref up to 256 characters"),
+        },
+    )
+
+    producer_ref = forms.CharField(
+        label=_("Producer ref (optional)"),
+        required=False,
+        validators=[length_validator],
+        help_text=_("Participant reference that identifies the producer of the data"),
+        error_messages={
+            "invalid": _("Enter a producer ref up to 256 characters"),
+        },
+    )
+
+    origin_ref = forms.CharField(
+        label=_("Origin ref (optional)"),
+        required=False,
+        validators=[length_validator],
+        help_text=_("The ATCO code which identifies the origin of the journey"),
+        error_messages={
+            "invalid": _("Enter a origin ref up to 256 characters"),
+        },
+    )
+
+    destination_ref = forms.CharField(
+        label=_("Destination ref (optional)"),
+        required=False,
+        validators=[length_validator],
+        help_text=_("The ATCO code which identifies the destination of the journey"),
+        error_messages={
+            "invalid": _("Enter a destination ref up to 256 characters"),
+        },
+    )
+
+    def __init__(self, instance=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def get_layout(self):
+        submit_button = ButtonSubmit("submit", "submit", content=_("Subscribe"))
+        cancel_button = LinkButton(
+            reverse(
+                "api:buslocation-api",
+                kwargs={},
+                host=config.hosts.DATA_HOST,
+            ),
+            content=_("Cancel"),
+            field_classes="govuk-button govuk-button--secondary",
+        )
+
+        return Layout(
+            Field("name"),
+            Field("url"),
+            HTML(
+                '<fieldset class="govuk-fieldset" role="group" aria-describedby="dataset-id-hint">'
+                '<legend class="govuk-fieldset__legend govuk-fieldset__legend--m"><h2 class="govuk-fieldset__heading">Dataset ID</h2></legend>'
```

Comment:
I wonder if we should update the naming of this field to "Data feed ID" as this is what BODS calls it when you go on "Browse location data"
![image](https://github.com/user-attachments/assets/505e0945-6297-4b3a-85b2-274489d77c38)
```suggestion
                '<legend class="govuk-fieldset__legend govuk-fieldset__legend--m"><h2 class="govuk-fieldset__heading">Data feed ID</h2></legend>'
```